spec/java_buildpack/jre/ibm_jre_initializer_spec.rb

```diff
@@ -0,0 +1,57 @@
+# Cloud Foundry Java Buildpack
+# Copyright 2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+require 'component_helper'
+require 'java_buildpack/component/mutable_java_home'
+require 'java_buildpack/jre/ibm_jre_initializer'
+
+describe JavaBuildpack::Jre::IbmJreInitializer do
```

Comment:
Can you please add a bit more whitespace to this file to better match [other examples](https://github.com/cloudfoundry/java-buildpack/blob/master/spec/java_buildpack/jre/open_jdk_like_jre_spec.rb#L21)?